core/src/main/java/org/hibernate/search/jsr352/internal/steps/lucene/EntityReader.java

```diff
@@ -177,7 +177,7 @@ public void open(Serializable checkpointId) throws Exception {
 
 		final int partitionId = Integer.parseInt( partitionIdStr );
 
-		LOGGER.debugf( "[partitionId=%d] open reader for entity %s ...", partitionId, entityName );
+		LOGGER.debugf( "[partitionId=%d] open reader for entity %s ...", (Integer) partitionId, entityName );
```

Comment:
The variable `partitionId` is an `int`,  parsed from `partitionIdStr`. Why should we box `int` to `Integer` before giving it to the logger? In [Formatter (Java Platform SE 7)](http://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html), paragraph _Conversions_: 

> 3.1 Integral - may be applied to Java integral types: byte, Byte, short, Short, int and Integer, long, Long, and BigInteger.

So I guess `int` should be fine.